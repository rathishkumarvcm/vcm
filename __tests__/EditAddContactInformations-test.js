import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import EditAddContactInformations from '../App/Screens/EditAddressAddNew/EditAddressAddNewComponent';

const navigation = { navigate: jest.fn(), getParam: jest.fn()};

describe('Add Contact Information', () => {

    // Application Screnshot
    it('snap shot', () => {
        const tree = renderer.create(<EditAddContactInformations navigation={navigation} />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    // Application Render
    test('render',()=>{
        const component = shallow(<EditAddContactInformations navigation={navigation} />);  
        component.render();
      });
});