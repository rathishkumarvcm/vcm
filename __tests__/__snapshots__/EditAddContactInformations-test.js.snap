// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Contact Information snap shot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F7FAFF",
      "flex": 1,
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    clickable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flex": 0.12,
        "flexDirection": "row",
        "opacity": 1,
        "paddingLeft": "4%",
        "paddingRight": "4%",
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": 100.5,
          "opacity": 1,
          "width": "25%",
        }
      }
    >
      <Image
        resizeMode="contain"
        source={
          Object {
            "testUri": "../../../App/Images/logo.png",
          }
        }
        style={
          Object {
            "alignItems": "flex-start",
            "justifyContent": "flex-start",
            "width": "100%",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "width": "12%",
        }
      }
    />
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "marginTop": 24.5,
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 40,
            "width": 56,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 40,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "EvilIcons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "width": "10%",
        }
      }
    />
    <View
      accessible={true}
      clickable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 30,
            "width": 44,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 30,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 30,
            "width": 44,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 30,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      clickable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "flex-end",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <RNGestureHandlerButton
        collapsable={false}
        onGestureEvent={[Function]}
        onGestureHandlerEvent={[Function]}
        onGestureHandlerStateChange={[Function]}
        onHandlerStateChange={[Function]}
        rippleColor={0}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            undefined,
          ]
        }
      >
        <View
          accessible={true}
          style={
            Object {
              "alignItems": "flex-end",
              "justifyContent": "center",
              "marginEnd": 10,
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "height": 40,
                "width": 56,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 40,
                  },
                  Object {
                    "backgroundColor": "transparent",
                    "marginHorizontal": 5,
                  },
                  Object {
                    "fontFamily": "Material Icons",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              testID="iconIcon"
            >
              
            </Text>
          </View>
        </View>
      </RNGestureHandlerButton>
    </View>
  </View>
  <RCTScrollView
    style={
      Object {
        "flex": 0.85,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 29.5,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#B2B2B2",
              "fontSize": 26.5,
            }
          }
        >
          Settings &gt; Personal info &gt; 
        </Text>
        <Text
          style={
            Object {
              "color": "#B2B2B2",
              "fontSize": 26.5,
              "fontWeight": "bold",
            }
          }
        >
          Edit Contact Information
        </Text>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 29.5,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#707070",
              "fontSize": 29.5,
              "fontWeight": "bold",
              "width": "70%",
            }
          }
        >
          Edit Contact Information
        </Text>
      </View>
      <View
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#B2B2B2",
            "marginLeft": "4%",
            "marginRight": "4%",
            "marginTop": 16.5,
          }
        }
      />
      <View
        style={
          Object {
            "flexDirection": "column",
            "marginTop": 33,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#6F7072",
              "fontSize": 33,
              "fontWeight": "bold",
            }
          }
        >
          Mailing Address
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#D4D4D4",
            "borderWidth": 1,
            "marginLeft": "2%",
            "marginRight": "2%",
            "marginTop": 33,
            "padding": "2%",
            "width": "96%",
          }
        }
      >
        <View
          style={
            Object {
              "marginTop": 29.5,
              "paddingLeft": "4%",
              "paddingRight": "4%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#333333DE",
                "fontSize": 26.5,
                "fontWeight": "bold",
                "marginBottom": "4%",
              }
            }
          >
            Address Type
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "marginBottom": "2%",
              "paddingLeft": "2%",
              "paddingRight": "2%",
              "width": "100%",
            }
          }
        >
          <View
            accessible={true}
            clickable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "flexDirection": "row",
                "marginBottom": "2%",
                "marginTop": "4%",
                "opacity": 1,
                "width": "92%",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "borderColor": "#707070",
                  "borderRadius": 26.5,
                  "borderWidth": 3.5,
                  "height": 52.5,
                  "justifyContent": "center",
                  "width": 52.5,
                }
              }
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#707070",
                    "borderRadius": 10,
                    "height": 19.5,
                    "width": 19.5,
                  }
                }
              />
            </View>
            <View
              style={
                Array [
                  Object {
                    "flexDirection": "column",
                    "height": 82,
                    "marginLeft": "5%",
                  },
                  Object {
                    "justifyContent": "center",
                  },
                ]
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                  }
                }
              >
                U.S. or U.S. Territories
              </Text>
              <Text
                style={
                  Object {
                    "alignItems": "flex-start",
                    "color": "#333333DE",
                    "fontSize": 19.5,
                    "marginTop": "5%",
                  }
                }
              />
            </View>
          </View>
          <View
            accessible={true}
            clickable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "flexDirection": "row",
                "marginBottom": "2%",
                "marginTop": "4%",
                "opacity": 1,
                "width": "92%",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "borderColor": "#707070",
                  "borderRadius": 26.5,
                  "borderWidth": 3.5,
                  "height": 52.5,
                  "justifyContent": "center",
                  "width": 52.5,
                }
              }
            />
            <View
              style={
                Array [
                  Object {
                    "flexDirection": "column",
                    "height": 82,
                    "marginLeft": "5%",
                  },
                  Object {
                    "justifyContent": "center",
                  },
                ]
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                  }
                }
              >
                APO (Army or Air Force Post Office)
              </Text>
              <Text
                style={
                  Object {
                    "alignItems": "flex-start",
                    "color": "#333333DE",
                    "fontSize": 19.5,
                    "marginTop": "5%",
                  }
                }
              />
            </View>
          </View>
          <View
            accessible={true}
            clickable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "flexDirection": "row",
                "marginBottom": "2%",
                "marginTop": "4%",
                "opacity": 1,
                "width": "92%",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "borderColor": "#707070",
                  "borderRadius": 26.5,
                  "borderWidth": 3.5,
                  "height": 52.5,
                  "justifyContent": "center",
                  "width": 52.5,
                }
              }
            />
            <View
              style={
                Array [
                  Object {
                    "flexDirection": "column",
                    "height": 82,
                    "marginLeft": "5%",
                  },
                  Object {
                    "justifyContent": "center",
                  },
                ]
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                  }
                }
              >
                FPO (Fleet Post Office)
              </Text>
              <Text
                style={
                  Object {
                    "alignItems": "flex-start",
                    "color": "#333333DE",
                    "fontSize": 19.5,
                    "marginTop": "5%",
                  }
                }
              />
            </View>
          </View>
          <View
            accessible={true}
            clickable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "flexDirection": "row",
                "marginBottom": "2%",
                "marginTop": "4%",
                "opacity": 1,
                "width": "92%",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "borderColor": "#707070",
                  "borderRadius": 26.5,
                  "borderWidth": 3.5,
                  "height": 52.5,
                  "justifyContent": "center",
                  "width": 52.5,
                }
              }
            />
            <View
              style={
                Array [
                  Object {
                    "flexDirection": "column",
                    "height": 82,
                    "marginLeft": "5%",
                  },
                  Object {
                    "justifyContent": "center",
                  },
                ]
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                  }
                }
              >
                DPO (Diplomatic Post Office)
              </Text>
              <Text
                style={
                  Object {
                    "alignItems": "flex-start",
                    "color": "#333333DE",
                    "fontSize": 19.5,
                    "marginTop": "5%",
                  }
                }
              />
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "marginTop": 29.5,
              "paddingLeft": "4%",
              "paddingRight": "4%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#333333DE",
                "fontSize": 26.5,
                "fontWeight": "bold",
                "marginBottom": "4%",
              }
            }
          >
            Address Line 1
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "marginBottom": "2%",
              "paddingLeft": "2%",
              "paddingRight": "2%",
              "width": "100%",
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#FFFFFF",
                  "borderColor": "#DEDEDF",
                  "borderRadius": 6.5,
                  "borderWidth": 1,
                  "height": 79,
                  "justifyContent": "center",
                  "width": "92%",
                },
                Object {
                  "flexDirection": "column",
                  "justifyContent": "center",
                  "marginBottom": "2%",
                  "paddingLeft": "2%",
                  "paddingRight": "2%",
                  "width": "100%",
                },
              ]
            }
          >
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              autoCorrect={false}
              autoFocus={false}
              editable={true}
              errorFlag={false}
              errorText=""
              inputStyle={Object {}}
              keyboardType="default"
              maxLength={50}
              multiline={false}
              numberOfLines={1}
              onBlur={null}
              onChange={null}
              onChangeText={[Function]}
              onSubmitEditing={null}
              placeholder="Address Line 1"
              placeholderTextColor={null}
              propInputStyle={
                Object {
                  "flexDirection": "column",
                  "justifyContent": "center",
                  "marginBottom": "2%",
                  "paddingLeft": "2%",
                  "paddingRight": "2%",
                  "width": "100%",
                }
              }
              rejectResponderTermination={true}
              returnKeyType="next"
              selectionColor={null}
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "color": "#B5B5B5",
                    "fontSize": 26.5,
                    "justifyContent": "center",
                    "marginLeft": 23,
                  },
                  Object {},
                ]
              }
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Object {
              "marginTop": 29.5,
              "paddingLeft": "4%",
              "paddingRight": "4%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#333333DE",
                "fontSize": 26.5,
                "fontWeight": "bold",
                "marginBottom": "4%",
              }
            }
          >
            Address Line 2
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "marginBottom": "2%",
              "paddingLeft": "2%",
              "paddingRight": "2%",
              "width": "100%",
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#FFFFFF",
                  "borderColor": "#DEDEDF",
                  "borderRadius": 6.5,
                  "borderWidth": 1,
                  "height": 79,
                  "justifyContent": "center",
                  "width": "92%",
                },
                Object {
                  "flexDirection": "column",
                  "justifyContent": "center",
                  "marginBottom": "2%",
                  "paddingLeft": "2%",
                  "paddingRight": "2%",
                  "width": "100%",
                },
              ]
            }
          >
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              autoCorrect={false}
              autoFocus={false}
              editable={true}
              errorFlag={false}
              errorText=""
              inputStyle={Object {}}
              keyboardType="default"
              maxLength={50}
              multiline={false}
              numberOfLines={1}
              onBlur={null}
              onChange={null}
              onChangeText={[Function]}
              onSubmitEditing={null}
              placeholder="Address Line 2"
              placeholderTextColor={null}
              propInputStyle={
                Object {
                  "flexDirection": "column",
                  "justifyContent": "center",
                  "marginBottom": "2%",
                  "paddingLeft": "2%",
                  "paddingRight": "2%",
                  "width": "100%",
                }
              }
              rejectResponderTermination={true}
              returnKeyType="next"
              selectionColor={null}
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "color": "#B5B5B5",
                    "fontSize": 26.5,
                    "justifyContent": "center",
                    "marginLeft": 23,
                  },
                  Object {},
                ]
              }
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Object {
              "marginTop": 29.5,
              "paddingLeft": "4%",
              "paddingRight": "4%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#333333DE",
                "fontSize": 26.5,
                "fontWeight": "bold",
                "marginBottom": "4%",
              }
            }
          >
            ZIP Code
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "column",
              "justifyContent": "center",
              "marginBottom": "2%",
              "paddingLeft": "2%",
              "paddingRight": "2%",
              "width": "100%",
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#FFFFFF",
                  "borderColor": "#DEDEDF",
                  "borderRadius": 6.5,
                  "borderWidth": 1,
                  "height": 79,
                  "justifyContent": "center",
                  "width": "92%",
                },
                Object {
                  "flexDirection": "column",
                  "justifyContent": "center",
                  "marginBottom": "2%",
                  "paddingLeft": "2%",
                  "paddingRight": "2%",
                  "width": "100%",
                },
              ]
            }
          >
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              autoCorrect={false}
              autoFocus={false}
              editable={true}
              errorFlag={false}
              errorText=""
              inputStyle={Object {}}
              keyboardType="number-pad"
              maxLength={5}
              multiline={false}
              numberOfLines={1}
              onBlur={[Function]}
              onChange={null}
              onChangeText={[Function]}
              onSubmitEditing={null}
              placeholder="ZIP Code"
              placeholderTextColor={null}
              propInputStyle={
                Object {
                  "flexDirection": "column",
                  "justifyContent": "center",
                  "marginBottom": "2%",
                  "paddingLeft": "2%",
                  "paddingRight": "2%",
                  "width": "100%",
                }
              }
              rejectResponderTermination={true}
              returnKeyType="next"
              selectionColor={null}
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "color": "#B5B5B5",
                    "fontSize": 26.5,
                    "justifyContent": "center",
                    "marginLeft": 23,
                  },
                  Object {},
                ]
              }
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "column",
              "marginTop": 33,
              "paddingLeft": "4%",
              "paddingRight": "4%",
              "width": "100%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#333333DE",
                "fontSize": 29.5,
                "fontWeight": "bold",
                "margin": "2%",
              }
            }
          >
            City
          </Text>
          <Text
            style={
              Object {
                "color": "#56565A",
                "fontSize": 29.5,
                "margin": "2%",
              }
            }
          >
            
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "column",
              "marginTop": 33,
              "paddingLeft": "4%",
              "paddingRight": "4%",
              "width": "100%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#333333DE",
                "fontSize": 29.5,
                "fontWeight": "bold",
                "margin": "2%",
              }
            }
          >
            State/Territory
          </Text>
          <Text
            style={
              Object {
                "color": "#56565A",
                "fontSize": 29.5,
                "margin": "2%",
              }
            }
          >
            
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
            "marginTop": 33,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderColor": "#56565A",
              "borderRadius": 1,
              "borderWidth": 1,
              "height": 82,
              "justifyContent": "center",
              "marginLeft": "4%",
              "marginRight": "4%",
              "marginTop": 19.5,
              "opacity": 1,
              "width": "92%",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                },
              ]
            }
          >
            Cancel
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
            "marginTop": 33,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#56565A",
              "borderColor": "#56565A",
              "borderRadius": 1,
              "borderWidth": 1,
              "height": 82,
              "justifyContent": "center",
              "marginLeft": "4%",
              "marginRight": "4%",
              "marginTop": 19.5,
              "opacity": 1,
              "width": "92%",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#FFFFFF",
                  "fontSize": 26.5,
                },
              ]
            }
          >
            Save
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#B2B2B2",
            "marginTop": "5%",
            "width": "100%",
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "padding": 24.5,
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 29.5,
              "marginBottom": 8,
            }
          }
        >
          Investing involves risk including loss of principal.
        </Text>
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 26.5,
              "justifyContent": "center",
            }
          }
        >
          Victory Mutual Funds and USAA Mutual Funds are distributed by Victory Capital Advisers, Inc. (VCA). VictoryShares ETFs and VictoryShares USAA ETFs are distributed by Foreside Fund Services, LLC (Foreside). VCA and Foreside are members of FINRA and SIPC. Victory Capital Management Inc. (VCM) is the investment adviser to the Victory Mutual Funds, USAA Mutual Funds, VictoryShares ETFs and VictoryShares USAA ETFs. VCA and VCM are not affiliated with Foreside. USAA is not affiliated with Foreside, VCM, or VCA. USAA and the USAA logos are registered trademarks and the USAA Mutual Funds and USAA Investments logos are trademarks of United Services Automobile Association and are being used by Victory Capital and its affiliates under license. Victory Capital means Victory Capital Management Inc., the investment manager of the USAA 529 College Savings Plan (Plan). The Plan is distributed by Victory Capital Advisers, Inc., a broker dealer registered with FINRA and an affiliate of Victory Capital. Victory Capital and its affiliates are not affiliated with United Services Automobile Association or its affiliates. USAA and the USAA logo are registered trademarks and the USAA 529 College Savings Plan logo is a trademark of United Services Automobile Association and are being used by Victory Capital and its affiliates under license.
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "flexDirection": "column",
            "height": 164.5,
            "paddingBottom": "5%",
            "paddingLeft": "5%",
          }
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../App/Images/logo.png",
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "flexDirection": "row",
            "paddingLeft": "5%",
            "paddingRight": "5%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 29.5,
            }
          }
        >
          Connect with Us
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "flexDirection": "row",
            "paddingLeft": "5%",
            "paddingRight": "5%",
          }
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../App/Images/twitterlogo.png",
            }
          }
          style={
            Object {
              "height": 40,
              "margin": "4%",
              "resizeMode": "contain",
              "width": 40,
            }
          }
        />
        <Image
          source={
            Object {
              "testUri": "../../../App/Images/linkedinlogo.png",
            }
          }
          style={
            Object {
              "height": 40,
              "margin": "4%",
              "resizeMode": "contain",
              "width": 40,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "flexDirection": "row",
            "height": 82,
            "justifyContent": "center",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#5D83AE",
              "fontSize": 26.5,
              "paddingLeft": "4%",
              "width": "50%",
            }
          }
        >
          Privacy Policy
        </Text>
        <Text
          style={
            Object {
              "color": "#5D83AE",
              "fontSize": 26.5,
              "paddingLeft": "4%",
              "width": "50%",
            }
          }
        >
          Fund Documents
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "flexDirection": "row",
            "height": 82,
            "justifyContent": "center",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#5D83AE",
              "fontSize": 26.5,
              "paddingLeft": "4%",
              "width": "50%",
            }
          }
        >
          User Agreement
        </Text>
        <Text
          style={
            Object {
              "color": "#5D83AE",
              "fontSize": 26.5,
              "paddingLeft": "4%",
              "width": "50%",
            }
          }
        >
          Support
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#56565A",
            "height": 82,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Copyright Victory Capital Management Inc. ©2020
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
